### PR TITLE
Checking existence of #question-form before calling functions

### LINKED ISSUE #183: Fix `Uncaught TypeError: Cannot read property 'checked' of undefined` bug
Appears on `certifications/janice-snuffy/start` and is breaking other JS on the page.

Likely caused by `$("input[name='13_RECORDS_TO_BE_FORWARDED_TO_BOARD_OF_VETERANS_APPEALS_OTHER']")[0].checked`

Need to check whether `input[name='13_RECORDS_TO_BE_FORWARDED_TO_BOARD_OF_VETERANS_APPEALS_OTHER']` exists on the page, or try:

    $("input[name='13_RECORDS_TO_BE_FORWARDED_TO_BOARD_OF_VETERANS_APPEALS_OTHER']")[0].attr('checked')

### PR DESCRIPTION
**Pull request by webinista**
closes #183
modified:   app/assets/javascripts/application.js

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -89,19 +89,21 @@ var questionsSubmit = function (event) {
...
-    $("#question-form").submit(questionsSubmit);
-
-    // Restore the hidden fields that should be displayed, used when back button is used
-    display_8b_explanation();
-    display_9b();
-    display_10c();
-    display_11b();
-    display_13_other();
+    if(!!$("#question-form").length) {
+        $("input[name='8B_POWER_OF_ATTORNEY']").on(display_8b_explanation);
```
Comment (alex) [inline app/assets/javascripts/application.js]: Why was this changed from `.change` to `on`?

**Comment by alex** [comment]
:+1: lgtm, will merge once tests pass

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (webinista) [inline app/assets/javascripts/application.js]: Should have been changed to `on('change'` which is in line with the latest jQuery API. Changed it back to `change`.
